src/App.js

```diff
@@ -50,13 +49,16 @@ function App() {
           foundWinningPattern = false;
         }
       });
-
-      if (foundWinningPattern) {
-        setResult({ winner: player, state: "Won" });
-      }
+      winningPattern(foundWinningPattern);
     });
   };
 
+  Const ;winningPattern = (foundWinningPattern) => {
```

Comment:
Does this code work? It looks like there is a typo and also it doesn't seem like `player` is passed into the function.

Reply:
The code does not work and I am debugging several errors displayed on the UI. Also, after removing the semicolon from Const winningPattern, I get an expected error. You mentioned player should be passed into the function?
My thought process was to split the function ‘checkWin’ into 2 functions – ‘winningPattern’ is the second function – 'winningPattern' is responsible for updating the state of result.

Reply:
It should be `const winningPattern =  ...`

Why do you think `Const ; winningPattern = ...` isn't throwing an error? but `Const winningPattern = ...` is?

Reply:
Thanks! — Because I used an Uppercase instead of a lowercase c.